<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BDC6-D-506R2MK" name="Default test" tests="1" failures="0" timestamp="7 Jun 2016 05:00:41 GMT" time="0.126" errors="0">
  <testcase name="@BeforeClass before" time="0.0" classname="Desktop.Attach_Form">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="-1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: -1
	at java.util.ArrayList.elementData(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at java.util.Collections$SynchronizedList.get(Unknown Source)
	at com.accenture.omnichannelframework.api.Adapters.getAdaptersMap(Adapters.java:159)
	at com.accenture.omnichannelframework.api.Adapters.getInstanceFromAdapter(Adapters.java:55)
	at com.accenture.omnichannelframework.api.OmnichannelFramework.getInstanceFromAdapter(OmnichannelFramework.java:73)
	at Desktop.Attach_Form.before(Attach_Form.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:517)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:140)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:170)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:104)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
    </failure>
  </testcase> <!-- @BeforeClass before -->
  <testcase name="uplaod_Generate_Forms" time="0.006" classname="Desktop.Attach_Form">
    <skipped/>
  </testcase> <!-- uplaod_Generate_Forms -->
</testsuite> <!-- Default test -->
